cmd/massa-wallet/thyra-plugin-wallet.go

```diff
@@ -44,11 +35,35 @@ func StartServer(app *fyne.App, port int) {
 	server := restapi.NewServer(massaWalletAPI)
 	server.ConfigureAPI()
 
-	// Set the port to listen on to the passed-in port
-	server.Port = port
+	ln, err := server.HTTPListener()
+	if err != nil {
+		log.Fatalln(err)
+	}
+	registerPlugin(ln)
 
 	if err := server.Serve(); err != nil {
-
 		log.Fatalln(err)
 	}
 }
+
+func registerPlugin(ln net.Listener) {
+	//nolint:gomnd
+	if len(os.Args) < 2 {
+		panic("this program must be run with correlation id argument!")
+	}
+
+	pluginID := os.Args[1]
+
+	standaloneMode := false
+
+	if len(os.Args) == 3 && os.Args[2] == "--standalone" {
```

Comment:
This flag is not describe. Furthermore I think that it should be an env variable instead.